### PR TITLE
Fix find inactives

### PR DESCRIPTION
**Pull request by ChuckL**
Rather than hammering Tumblr's api to find last updated blogs, we're doing a more course analysis with the data tumblr has on followee's row.

Also fixed the progress bar.

### CODE REVIEW

**Comment by ChuckL** [comment]
This kind of fixes #255
I'll be working on making it even faster.

File: Extensions/find_inactives.js [hunk 1]
```diff
@@ -24,8 +24,8 @@ XKit.extensions.find_inactives = new Object({
 				"A month", "30",
 				"Two months", "60",
 				"4 months", "120",
-				"A year", "365",
-			],
```
Comment (nightpool) [inline Extensions/find_inactives.js]: why remove the commas?

Comment (ChuckL) [inline Extensions/find_inactives.js]: The commas were just unneeded and my IDE nagging me about them.  If it's part of the style we want to keep I can put them back in.

**Comment by ChuckL** [comment]
Updated the code so it would handle having more than 1 request at a time.

**Comment by ThePsionic** [comment]
Tested, looks good to me.
@homu r+

**Pull request merged**